.github/workflows/build.yaml

```diff
@@ -0,0 +1,49 @@
+name: Continuous Integration
+
+on:
+    pull_request: ~
+    push: ~
+
+jobs:
+    tests:
+        runs-on: ubuntu-20.04
+        strategy:
+            max-parallel: 1
+            matrix:
+                php:
+                    - '7.2'
+                    - '7.3'
+                    - '7.4'
+                    - '8.0'
```

Comment:
the normal jobs must have a value set for `description` or `composer_option`. Otherwise, they are swallowed by included jobs which are a super-set of their values.

Reply:
Are you sure? [Docs](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#example-including-new-combinations) say "Any unmatched include configurations are added to the matrix"

Reply:
yeah, but if you only add new fields which are not part of the normal matrix, this will be replace the expected normal job, because the `php: '7.2'` job will be matched by the `php: '7.2', composer_option: '--prefer-lowest` one.

Reply:
btw, if you look at https://github.com/garak/KnpMenuBundle/actions/runs/485202488, you'll see it only has 4 jobs in the matrix, proving my point.

Reply:
Removing all jobs running a normal `composer update` was not the right fix for that. These jobs are important.

Reply:
So, should we re-add `jobs` key? With a description for each entry?

Reply:
you should re-add them, with `composer_option` using an empty string, `dev` using `false`, etc...